### ISSUE TITLE
HighlightAuto doesn't work with provided examples

### ISSUE THREAD

**Issue by WardPearce** [metonym/svelte-highlight#197]
Greetings,
Appears `HighlightAuto` doesn't work even with provided examples.

![screenshot](https://i.imgur.com/Kqpr4PM.png)

Doesn't work in my local environment too.
Thanks,
Ward

**Comment by WardPearce** [comment]
```ts
  import hljs from 'highlight.js'
  hljs.initHighlighting()
  ```
  Adding the above after importing `import { HighlightAuto } from 'svelte-highlight'` resolved the issue.

**Comment by metonym** [comment]
Thank you for raising this issue. I'm able to reproduce it. Strangely, [`HighlightAuto` works on the SvelteKit demo site](https://github.com/metonym/svelte-highlight/blob/ca3f390e1d4d974b3b983778c9a2343edaf650f8/demo/routes/index.svelte#L223).

It seems that using `hljs` from the core library is not enough. It needs to load all grammars to guess the language.

**Comment by metonym** [comment]
Fixed in v5.3.1

**Comment by metonym** [comment]
Note: you'll also need to add "highlight.js" to `vite.optimizeDeps.include` if you are using a SvelteKit or vite set-up:

```js
// svelte.config.js
export default {
  kit: {
    adapter: adapter(),
    vite: {
      optimizeDeps: {
        include: ["highlight.js", "highlight.js/lib/core"],
      },
    },
  },
};
```

Sample usage:

```svelte
<script>
  import { HighlightAuto } from "svelte-highlight";
  import github from "svelte-highlight/src/styles/github";

  $: code = `body {\n  padding: 0;\n  color: red;\n}`;
</script>

<svelte:head>
  {@html github}
</svelte:head>

<HighlightAuto {code} />
```

Result:

<img width="518" alt="Screen Shot 2022-03-16 at 8 08 31 PM" src="https://user-images.githubusercontent.com/10718366/158728804-5793bcd5-5fe0-4311-9691-269e709968f1.png">
